src/scheduler/node_info.c

```diff
@@ -2968,6 +2971,11 @@ eval_simple_selspec(status *policy, chunk *chk, node_info **pninfo_arr,
 
 	cur_flt_lic = flt_lic;
 
+	aoename = resresv->aoename;
+	aoereq = find_resource_req(specreq_noncons, aoedef);
+	if (aoereq == NULL && !pl->pack)
+		resresv->aoename = NULL;
```

Comment:
Why not use aoename here since you just set it to resresv->aoename?

Reply:
I have modified the code, so this will not be needed anymore.